bytes/equals_test.ts

```diff
@@ -19,7 +19,7 @@ Deno.test("equals() handles randomized testing", () => {
     const arr3 = arr1.slice(0);
     // the chance of arr1 equaling arr2 is basically 0
     // but introduce an inequality at the end just in case
-    arr2[arr2.length - 1] = arr1[arr1.length - 1] ^ 1;
+    arr2[arr2.length - 1] = arr1.at(-1)! ^ 1;
```

Comment:
`arr1` is a `Uint8Array` with a length between 995 and 1004, so it will never be empty and `.at(-1)` will always be in bounds.